service/controller/app/v1/resource/chart/desired.go

```diff
@@ -106,6 +109,31 @@ func hasSecret(cr v1alpha1.App, appCatalog v1alpha1.AppCatalog) bool {
 	return false
 }
 
+func replacePrefix(from string) string {
+	return strings.Replace(from, "app-operator.giantswarm.io", "chart-operator.giantswarm.io", 1)
```

Comment:
Used `ReplaceAll` but failed in CircleCI that function was introduced from 1.12.

Reply:
Damn thanks for trying and we should move to Go 1.12 at some point.